easysocial/easysocial/message.php

```diff
@@ -60,7 +60,7 @@ public function post()
 	 *
 	 * @since 1.0
 	 */
-	public function newMessage()
+	private function newMessage()
 	{
 		$app			=	JFactory::getApplication();
 		$recipients		=	$app->input->get('recipients', null, 'ARRAY');
```

Comment:
@Pratikshakhandagale - set default value as blank array as expected format is ARRAY